packages/valory/skills/abstract_round_abci/models.py

```diff
@@ -743,7 +718,8 @@ class BenchmarkTool(Model, FrozenMixin):
     def __init__(self, *args: Any, **kwargs: Any) -> None:
         """Benchmark tool for rounds behaviours."""
         self.benchmark_data = {}
-        self.log_dir = Path(kwargs.pop("log_dir", "/logs"))
```

Comment:
default args should never be specified for these params in code, use skill.yaml instead